### DISCUSSION TITLE
[ASK] Teler won't build

### DISCUSSION THREAD

**Discussion by pieterhouwen** [teler-sh/teler#174] [Q&A] [answered]
**Describe the bug**

Cannot run make build. Exits with following errors:

![image](https://user-images.githubusercontent.com/10866686/128541875-aaca02ad-7386-4d35-8a44-7c374dad1b2f.png)

**To Reproduce**

On a fresh Ubuntu install, install golang, clone the repo and run make build.

```yaml
Your teler config file...
The default one.

// Please redacted your token and/ other sensitive informations
```

**Expected behavior**

It builds without errors

**Screenshots**

**Environment (please complete the following information):**

- OS: Ubuntu Bionic (18.04)
- OS version: Linux vpn-machine 4.15.0-151-generic #157-Ubuntu SMP Fri Jul 9 23:07:57 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
- teler Version N/A

**Additional context**
Add any other context about the problem here. Full output log is probably a helpful thing to add here.

- See screenshot

[a comment by an automated account is not shown]

**Comment by dwisiswant0** [comment]
What version of Go are you using, @pieterhouwen?

**Comment by dwisiswant0** [comment, marked as answer]
It looks like the version of Go you are using is version `1.10`.

The minimum version of Go must be **1.15**. Try upgrading and then rebuilding.
